data/settings/src/main/proto/com/google/jetpackcamera/settings/aspect_ratio.proto

```diff
@@ -0,0 +1,26 @@
+/*
+ * Copyright (C) 2023 The Android Open Source Project
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+syntax = "proto3";
+
+option java_package = "com.google.jetpackcamera.settings";
+option java_multiple_files = true;
+
+enum AspectRatioProto {
```

Comment:
Lets drop "Proto" from the name.

We should also rename the others later.

Reply:
we'll have a name conflict with the AspectRatio enum from `com.google.jetpackcamera.settings.model`, as well as another auto-generated class (from using proto datastore) in the build folder since we were using; 
I was intentional about keeping proto in the name to help differentiate from the enum equivalent we use for the rest of the app, see `LocalSettingsRepository.kt`